base.py

```
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
+    bl_label = "Some of the nodes/elements are missing"
+    
+    hide = bpy.props.BoolProperty()
```

do we really need two properties? If one chooses to delete the objects, it cannot hide them, and hiding an object to then delete it makes no sense, right?

---

I assumed the default option to be not do anything, neither hide the objects, nor delete them.

But if I understand you, you believe the objects should be hid by default, and only if the user wants it, should we delete them ? Am I right?

---

Uhm... No actually I like your solution, I just didn't understand it at the beginning. It's okay to let users keep the old objects in place, if they so desire.